### PR TITLE
Update SGX_0-base_Enclave.md

### PR DESCRIPTION
**Pull request by qinkunbao**

### CODE REVIEW

File: docs/DesignDocs/SGX_0-base_Enclave.md [hunk 1]
```diff
@@ -139,8 +139,8 @@ within.  For each 0-base Enclave within a process, the ECALL/OCALL marshalling
 buffer and any shared memory an Enclave needs to access must be allocated above
 the Enclave's `SECS.SIZE` value.
 
-Running multiple instance of a single 0-base Enclave in a single process won't
-be possible, as each instance would has to be loaded at exactly the same
+Running multiple instances of 0-base Enclave in a single process won't
```
Comment (mingweishih) [inline docs/DesignDocs/SGX_0-base_Enclave.md:142]:
```suggestion
Running multiple instances of a 0-base Enclave in a single process won't
```

**Review by RRathna — APPROVED** [review]
LGTM

File: docs/DesignDocs/SGX_0-base_Enclave.md (hunk 1 above)
Comment (qinkunbao) [reply docs/DesignDocs/SGX_0-base_Enclave.md]: Thanks. Done.

**Review by mingweishih — APPROVED** [review]
Thanks. Can you squash the commits?

**Comment by qinkunbao** [comment]
> Thanks. Can you squash the commits?

Done!

**Review by mingweishih — APPROVED** [review]

**Pull request merged**